data/drop-tables.js

```diff
@@ -0,0 +1,21 @@
+const pool = require('../lib/utils/pool');
+
+run();
+
+async function run() {
+
+  try {
+    await pool.connect();
+
+    await pool.query(`
+        DROP TABLE IF EXISTS links CASCADE;
```

Comment:
These two tables aren't joined, so I don't think CASCADE is necessary. But I'm not quite sure either way.

Reply:
Cool. Good point. Thanks!